usaspending_api/core/validator/helpers.py

```diff
@@ -25,11 +25,14 @@
 def _check_max(rule):
     value = rule['value']
     if rule['type'] in ('integer', 'float'):
-        if value > rule['max']:
+        if value > rule['max'] and rule['max'] != 0:
```

Comment:
Unsure it is a good idea to have a unenforced max value as 0. Would it be easier to apply the max defaults by type instead of setting the value to 0? They are already very large values and wouldn't require special logic.

Reply:
This is an artifact and should have been removed, good catch